index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title> Taher Haggui </title>
+        <link rel="stylesheet" type="text/css" href="styles/style.css">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+    </head>
+    <body>
+        <header>
+          <nav id="nav">
+            <a class="logo" href="#">My Logo</a>
+            
+            <nav><img src="statics/images/Union.png" alt="Menu"></nav>
+               
+          </nav>
+        </header>
+        <section>
+           <h1>I’m Taher
+            Glad to see you!</h1>
+            <p>
+                I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+            <footer>
+                <h3>LET'S CONNECT</h3>
+                <div>
+                   <ul>
+                     <li class="icon"><a href="#"><img src="statics/images/twitter.png" alt="twitter link"></a></li>
+                     <li class="icon"><a href="#"><img src="statics/images/Linkedin_icon.png" alt="linked link"></a></li>
```

Comment:
Original Design : 
![image](https://user-images.githubusercontent.com/43698511/133693704-f7cc03bd-0fa0-45ba-bb54-96e8a43dd192.png)
Your webpage : 
![image](https://user-images.githubusercontent.com/43698511/133693779-299cecc7-1f1a-43ea-97d2-2a1a28829904.png)

- [ ] As you can see from the pictures, your project looks a bit different from the original design. The icons should be aligned to the left and need more space between them. The logo and the menu icon need more space between them. The Headline section needs more margin-left. Kindly make this change respecting the original design is one of the project requirements as it is an important skill for a front-end developer